3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/adapters/registry/GoogleAnalyticsAdapter.scala

```diff
@@ -0,0 +1,662 @@
+/*
+ * Copyright (c) 2012-2017 Snowplow Analytics Ltd. All rights reserved.
```

Comment:
2018!

Reply:
That should be a dedicated commit cross-codebase (not file by file)

Reply:
Does it mean we have to put this commit for each first release in new year?

Reply:
Or this can go into the prep for rel commit since it's just housekeeping

Reply:
I added a couple of tickets just to be super-explicit (especially because not every component codebase will be modified in the first release of the year).